### PR TITLE
use openshift_bootstrap_autoapprover to approve bootstrap CSRs

### PR DESCRIPTION
**Pull request by jim-minter**
@kargakis @pweil- 

This works as follows (following direction from clayton):
- master node bootstrap kubeconfig is actually admin.kubeconfig
- 1st csr (master node client cert) is submitted and autoapproved by controller on basis of being submitted by system:admin
- 2nd csr (master node server cert) sits and waits
- this is good enough for the master node to register and become ready, although oc exec and oc logs wouldn't yet work
- ansible deploys the autoapprover pod which lands on the master
- the autoapprover picks up the 2nd master node csr and all the remaining node csrs and approves them
- remaining nodes become available for remaining workload

### CODE REVIEW

File: pkg/openshift/certgen/templates/master/tmp/ansible/azure-ocp-deploy.yml [hunk 1]
```diff
@@ -3,6 +3,10 @@
   gather_facts: false
```
Comment (kargakis) [inline pkg/openshift/certgen/templates/master/tmp/ansible/azure-ocp-deploy.yml]: Shouldn't the name of this playbook be updated?

**Comment by kargakis** [comment]
Nice, lgtm minus a question about the playbook name.

File: pkg/openshift/certgen/templates/master/tmp/ansible/azure-ocp-deploy.yml (hunk 1 above)
Comment (jim-minter) [reply pkg/openshift/certgen/templates/master/tmp/ansible/azure-ocp-deploy.yml]: Updated, thanks.

**Comment by jim-minter** [comment]
Updated; @jackfrancis or @CecileRobertMichon please merge.

**Pull request merged**